pyproject.toml

```diff
@@ -1,3 +1,55 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[project.urls]
+pypi = "https://pypi.org/project/httpbin/"
+Repository = "https://github.com/psf/httpbin"
+
+[project]
+name = "httpbin"
+version = "0.10.0"
+requires-python = ">=3.7"
+description = "HTTP Request and Response Service"
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
```

Comment:
We should add maintainers as well. This implies the author is still actively working on the project which they are not.

Reply:
Do you have a preference about who to add?

Reply:
I don't expect to have much time to spend on this so I'd ask @nateprewitt and @kevin1024 how they want to be referenced there

Reply:
Similarly, I can help with critical bugs/infra updates but I don't believe I'll have much time to dedicate beyond that. I'm fine being listed with Kevin if we need to put someone other than Kenneth as a contact.

Reply:
So if that's okay, I'll add Kenneth, Nate and Kevin to the list once I got feedback of @kevin1024.

Reply:
Kenneth should remain the author. They're not maintaining this to the best of my knowledge though and do not belong there

Reply:
Right. I did mean maintainers. Not authors. Sorry if that wasn't clear.